cache.rst

```diff
@@ -486,15 +486,15 @@ to be able to invalidate the cache more efficient. One way to achieve that is to
 use cache tags. One or more tags could be added to the cache item. All items with
 the same key could be invalidate with one function call::
 
-    use Symfony\Contracts\Cache\ItemInterface;
+    use Symfony\Contracts\Cache\TagAwareCacheInterface;
 
-    $value0 = $pool->get('item_0', function (ItemInterface $item) {
+    $value0 = $pool->get('item_0', function (TagAwareCacheInterface $item) {
```

Comment:
It is the `$pool` that should be a `TagAwareCacheInterface`. 

@pavelmaca could you update this PR a little? Could you wrap this code into a function or a class and make sure `$pool` is a `TagAwareCacheInterface`?